tsconfig.app.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
```

Comment:
**suggestion**: Consider updating the target to a more recent ECMAScript version if possible, to leverage newer JavaScript features.